protobuf/controller/health.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package controller;
+
+option go_package = "github.com/layer5io/meshkit/protobuf/controller;controller";
+
+enum ControllerStatus {
```

Comment:
Is it common to prepend entities with package name?

Reply:
Yes same as explained above, if we use `Status` here then any package that imports this one cannot have `Status` defined anywhere.